hbase-server/src/main/java/org/apache/hadoop/hbase/master/procedure/ReopenTableRegionsProcedure.java

```diff
@@ -61,20 +69,36 @@ public class ReopenTableRegionsProcedure
 
   private List<HRegionLocation> regions = Collections.emptyList();
 
+  private List<HRegionLocation> currentRegionBatch = Collections.emptyList();
+
   private RetryCounter retryCounter;
 
+  private final long reopenBatchBackoffMillis;
```

Comment:
Procedure should not have final fields which have to be initialized by input. As when loading the procedure from procedure store, we can only initialize them when calling deserialize method...

Reply:
Sounds good. I should probably add these fields to the state data proto too, right?

Reply:
I don't think so because they're just configuration. Probably preferable for the configuration to update on restart rather than be stuck on what it started with. Maybe see if other procedures take configurations like this